tools/deployment/make_linux_package.sh

```diff
@@ -217,8 +273,8 @@ function main() {
 
   EPILOG="--url https://osquery.io \
     -m osquery@osquery.io          \
-    --vendor Facebook              \
-    --license BSD                  \
+    --vendor $PACKAGE_VENDOR       \
```

Comment:
```suggestion
    --vendor \"$PACKAGE_VENDOR\"       \
```

Should probably be quoted